lib/config/types.ts

```diff
@@ -234,7 +234,11 @@ export type UpdateType =
   | 'rollback'
   | 'bump';
 
-export type MatchStringsStrategy = 'any' | 'recursive' | 'combination';
+export enum MatchStringsStrategy {
```

Comment:
Do not use enum or move to utils, otherwise typescript will hold a runtime reference to the types file which we like to avoid for perf.

Reply:
K, thx for the info. I have reverted the move to enum.